setup.py

```diff
@@ -196,6 +200,7 @@
     + HBASE_REQUIRED
     + CASSANDRA_REQUIRED
     + AZURE_REQUIRED
+    + ROCKSET_REQUIRED
```

Comment:
you should also add this down below to `extras_require`

this will allow people to install Feast with Rockset by doing `pip install feast[rockset]`